pyshelf/artifact_key_filter.py

```diff
@@ -107,15 +107,15 @@ def _is_metadata(path):
 
 
 def _is_not_private(path):
-    return not _is_private(path)
+    return not is_private(path)
 
 
-def _is_private(path):
+def is_private(path):
     # If it starts with an underscore or
     # any of the parts of the path start
     # with an underscore it is considered
     # private
-    return bool(re.search("^_|/_(?!metadata)", path))
+    return bool(re.search("^_(?!metadata)|/_(?!metadata)", path))
```

Comment:
actually now that I think about it this wont work for all situations. I will fix...